### DISCUSSION TITLE
Wrong `pos_` for "BATATA"

### DISCUSSION THREAD

**Discussion by mbyalsky-r7** [explosion/spaCy#12417] [Help: Model Advice] [answered]
## How to reproduce the behaviour
Wrong `pos_` for batata, get `ADJ` should be `PROPN`

```
In [21]: english_nlp = spacy.load("en_core_web_md")

In [22]: text_nlp = english_nlp("I want BATATA! He pleaded.")

In [23]: batata_nlp = text_nlp[2]

In [24]: batata_nlp.pos_
Out[24]: 'ADJ'

In [25]: batata_nlp
Out[25]: BATATA
```

## Your Environment
* Operating System: macOS-13.2
* Python Version Used: 3.10.9
* spaCy Version Used: 3.5.1
* Environment Information: poetry

**Comment by danieldk** [comment, marked as answer]
I have converted this to a discussion, since this is not a general issue with spaCy.

Models will always make prediction errors due to various factors, such as: limited training data, model size, model architecture, etc. There are various things you could do to improve tagging accuracy:

* Use a larger/more powerful model such as `en_core_news_lg`/`en_core_web_trf`. The accuracies for the English models are available at: https://spacy.io/models/en
* If you want to fix up some systematic errors, you could use the attribute ruler component: https://spacy.io/usage/linguistic-features#mappings-exceptions
* If you are working on data from a specific domain, you could train a custom model for that particular domain (provided that annotations are available).
